src/GLViewer.ts

```diff
@@ -2784,8 +2784,8 @@ export class GLViewer {
                 var u, v, w;
 
                 u = Math.cos(beta);
-                v = (Math.cos(alpha) - Math.cos(beta) * Math.cos(gamma)) / Math.sin(gamma);
-                w = Math.sqrt(Math.max(0, 1 - u * u - v * v));
```

Comment:
why is this changed?  It doesn't calculate the same value and removes the guard against generating a NaN.

Reply:
@dkoes Does u,v accept negative values?

Reply:
Why does that matter?